### ISSUE TITLE
Move archiver and git-branch to devDependencies?

### ISSUE THREAD

**Issue by hubgit** [IBM/plex#233]
It feels like `archiver` and `git-branch` shouldn't need to be installed as dependencies when using the font files.

https://github.com/IBM/plex/blob/master/package.json#L22-L25

**Comment by akx** [comment]
Agreed. Installing a font package for production probably shouldn't install 105 packages.
```
$ yarn add @ibm/plex
success Saved 105 new dependencies.
info Direct dependencies
└─ @ibm/plex@1.2.3
$
```

Related to #177 too.

**Comment by vpicone** [comment]
Going to have to look into this, the only reason I can see for having them as regular dependencies is some limitation of the CI build.

**Comment by mjabbink** [comment]
@vpicone should this close?
